src/main/java/org/sagebionetworks/web/client/presenter/ACTAccessApprovalsPresenter.java

```diff
@@ -58,58 +63,59 @@
 	boolean isLoadingAllData;
 	
 	@Inject
-	public ACTAccessApprovalsPresenter(final ACTAccessApprovalsView view, SynapseAlert synAlert, PortalGinInjector ginInjector, LoadMoreWidgetContainer loadMoreContainer, final Button showHideAccessRequirementButton, DataAccessClientAsync dataAccessClient, SynapseSuggestBox peopleSuggestWidget, UserGroupSuggestionProvider provider, UserBadge selectedUserBadge, AccessRequirementWidget accessRequirementWidget, GlobalApplicationState globalAppState) {
+	public ACTAccessApprovalsPresenter(final ACTAccessApprovalsView view, SynapseAlert synAlert, PortalGinInjector ginInjector, LoadMoreWidgetContainer loadMoreContainer, final Button showHideAccessRequirementButton, DataAccessClientAsync dataAccessClient, SynapseSuggestBox accessorSuggestWidget, SynapseSuggestBox submitterSuggestWidget, UserGroupSuggestionProvider provider, UserBadge selectedSubmitterUserBadge, UserBadge selectedAccessorUserBadge, AccessRequirementWidget accessRequirementWidget, GlobalApplicationState globalAppState) {
 		this.view = view;
 		this.synAlert = synAlert;
 		this.ginInjector = ginInjector;
 		this.dataAccessClient = dataAccessClient;
 		fixServiceEntryPoint(dataAccessClient);
 		this.loadMoreContainer = loadMoreContainer;
-		this.selectedUserBadge = selectedUserBadge;
+		this.selectedSubmitterUserBadge = selectedSubmitterUserBadge;
 		this.showHideAccessRequirementButton = showHideAccessRequirementButton;
 		this.accessRequirementWidget = accessRequirementWidget;
 		this.globalAppState = globalAppState;
-		this.peopleSuggestWidget = peopleSuggestWidget;
```

Comment:
Renamed peopleSuggestWidget to submitterSuggestWidget, and selectedUserBadge to selectedSubmitterUserBadge.  Added accessorSuggestWidget and selectedAccessorUserBadge